src/styles/styles.js

```diff
@@ -855,10 +855,11 @@ const styles = {
         backgroundColor: themeColors.buttonDefaultBG,
     },
 
-    autoGrowHeightInputContainer: (textInputHeight, maxHeight) => ({
-        height: textInputHeight >= maxHeight ? maxHeight : textInputHeight,
-        minHeight: variables.componentSizeLarge,
-    }),
+    autoGrowHeightInputContainer: (textInputHeight, maxHeight) => {
+        const minHeight = variables.componentSizeLarge;
+        const height = Math.min(Math.max(minHeight, textInputHeight), maxHeight);
```

Comment:
This can be more readable, I think the code was more self-explanatory in your original proposal
```js
let height = textInputHeight;
if (height >= maxHeight) {
    height = maxHeight;
} else if (height <= minHeight) {
    height = minHeight;
}
```

Reply:
Hi, will check shortly
Weird, there is no "Review request" on this PR for me, I wonder why it wasn't applied automatically